src/EditorFeatures/CSharpTest/MoveDeclarationNearReference/MoveDeclarationNearReferenceTests.cs

```diff
@@ -1326,6 +1326,133 @@ void Local<T>()
     }
 
     public static void Out<T>(out T t) => t = default;
+}");
```

Comment:
any chance this work could be broken out to make this PR smaller/simpler?

Reply:
Didn't you already review this months ago?

Reply:
(and in @CyrusNajmabadi's defense....with the number of PRs Roslyn gets it's very easy to forget what you have and haven't reviewed.)